SparkPost/support-docs
# WE-341: upgrading to nextjs 12

### What Changed

Upgraded next from 11 to 12. Upgraded to latest version of `react` and `react-dom`. Using the latest version of the netlify/next plugin

### How to verify
- Verify the build completes locally
- Verify that the build completes without errors
- Inspect the preview deploy link and verify everything looks good on the site.
